examples/Comtrya.yaml

```diff
@@ -10,3 +10,6 @@ manifests:
 variables:
   fkbr: "sxoe"
   kuci: "xhpv"
+
+variables_source:
```

Comment:
How about `import_variables`, sounds a little less programmer-y

Reply:
I'm suggesting this because this is the public API to Comtrya users and we don't want to get this wrong up front. All the code references can change, but i'd like this interface to feel natural to people.

Reply:
Ansible manifests use `include` for e.g. `include_tasks`. We could use it here `include_variables`.

But `import_variables` is also fine. I would rather go with `include`, because it happens every time.